google/cloud/forseti/scanner/scanner_builder.py

```diff
@@ -50,78 +50,71 @@ def __init__(self, global_configs, scanner_configs, service_config,
 
     def build(self):
         """Build the enabled scanners to run.
-
         Returns:
             list: Scanner instances that will be run.
         """
         runnable_scanners = []
-        # please note that the scanner name passed in CLI must match the
-        # corresponding key in this dictionary
-        long_running_scanner_factory = {
-            'external_project_access_scanner': (
-                external_project_access_scanner.ExternalProjectAccessScanner,
-                'external_project_access_rules.yaml')
-        }
-        if self.scanner_name in long_running_scanner_factory:
-            scanner = self._instantiate_scanner(
-                long_running_scanner_factory[self.scanner_name][0],
-                long_running_scanner_factory[self.scanner_name][1],
-            )
-            runnable_scanners.append(scanner)
+        if self.scanner_name:
+            # remove the _scanner from scanner name so that it matches the key
+            # in requirements map
+            scanner = self._instantiate_scanner(self.scanner_name[:-8])
```

Comment:
Can you please add a TODO to add a unit test later?

Reply:
done!